### DISCUSSION TITLE
Can i add credit/debit card payment on exo platform / open teknik community for membership tier management payments

### DISCUSSION THREAD

**Discussion by Yasmina-Fahmy** [swellstores/community#460] [Help] [answered]
I am building my own business community either using exo platform or open teknik and i need to include membership tier management module which allows me to take community subscription from  my customers . Open teknik only uses stripe or paypal which dont work in some countries like Egypt , i need to replace it with credit/debit card payments and add it on my community. is this possible? can someone help me to do so? Thanks

#219 #249 #162

**Comment by sags95** [comment, marked as answer]
Yes it is possible through what we call "manual payment methods". We do have a technical guide here that walks through how to accomplish this: https://developers.swell.is/guides/integrate-payment-gateways-methods

You would have to build the actual integration with the gateways/methods that you mentioned, we have a network of various technical partners if you're looking for support to build this out.

**Comment by Yasmina-Fahmy** [comment]
Hi Team

Thanks a lot for your reply.  I actually need your help in building the
actual integration with the gateways/methods mentioned.

Can you please support me in building this on open teknik.

Thanks a lot
